frontend/src/components/interviewCard.js

```diff
@@ -49,8 +59,12 @@ class InterviewCard extends Component {
               {this.props.interviewer}
             </p>
             <>
-              <Button onClick={() => this.handleViewDetails()}>View Details</Button>
-              <Button onClick={() => this.toggle()}>Delete (Directors Only)</Button>
+              <Button color="info" onClick={() => this.handleViewDetails()}>
+                View Details
+              </Button>
+              <Button color="danger" className="ml-3" onClick={() => this.toggle()}>
+                Delete (Directors Only)
```

Comment:
is it possible to disable or just not show this button unless you're a director? (this might be a more appropriate task for me while integrating the new auth stuff)

Reply:
yes, I was just going to wait until all the auth stuff to be fully finalized do things like that, just in case the way we get role data is different.